configcatclient/configcatclient.py

```diff
@@ -191,27 +192,28 @@ def get_key_and_value(self, variation_id):
         :param variation_id: variation ID
         :return: key and value
         """
-        config, _ = self.__get_config()
+        config, _ = self._get_config()
         if config is None:
             self.log.error('Config JSON is not present. Returning %s.', 'None', event_id=1000)
             return None
 
         settings = config.get(FEATURE_FLAGS, {})
         try:
             for key, value in list(settings.items()):
+                setting_type = value.get(SETTING_TYPE)
                 if variation_id == value.get(VARIATION_ID):
-                    return KeyValue(key, get_value(value))
+                    return KeyValue(key, get_value(value, setting_type))
```

Comment:
<3

(I missed adding this check to this method in the JS SDK. I will need to correct that.)